_dev/front/js/components/List/List.vue

```diff
@@ -27,51 +27,56 @@
         class="wishlist-list-item"
         :key="list.id_wishlist"
         v-for="list of items"
+        :class="{ 'wishlist-list-item-default': list.default }"
       >
         <a
-          class="wishlist-list-item-title"
-          :href="list.listUrl"
+          class="wishlist-list-item-link"
+          @click="redirectToList(list.listUrl)"
         >
-          {{ list.name }}
-          <span v-if="list.nbProducts">({{ list.nbProducts }})</span>
-          <span v-else>(0)</span>
-        </a>
-
-        <div class="wishlist-list-item-right">
-          <button
-            class="wishlist-list-item-actions"
-            @click="togglePopup(list.id_wishlist)"
-            v-if="!list.default"
-          >
-            <i class="material-icons">more_vert</i>
-          </button>
+          <p class="wishlist-list-item-title">
+            {{ list.name }}
+            <span v-if="list.nbProducts">({{ list.nbProducts }})</span>
+            <span v-else>(0)</span>
+          </p>
 
-          <button
-            @click="toggleShare(list.id_wishlist, list.shareUrl)"
-            v-if="list.default"
-          >
-            <i class="material-icons">share</i>
-          </button>
+          <div class="wishlist-list-item-right">
+            <button
+              class="wishlist-list-item-actions"
+              @click.stop="togglePopup(list.id_wishlist)"
```

Comment:
```suggestion
              @click.prevent="togglePopup(list.id_wishlist)"
```
Should be enough :thinking:

Reply:
Nop, it prevent the event sent by the element itself, not the event from the parent element